### PR TITLE
Added missing alt1, alt2 in BBox constructor, BBox put in README

### PR DESCRIPTION
**Pull request by armantorkzaban**

### CODE REVIEW

File: lib/src/geojson.dart [hunk 1]
```diff
@@ -317,12 +321,28 @@ class BBox extends CoordinateType {
...
   num? get alt2 => _is3D ? _items[5] : null;
 
+  BBox copyWith({
+    num? lng1,
+    num? lat1,
+    num? lng2,
+    num? lat2,
+    num? alt1,
+    num? alt2,
+  }) =>
```
Comment (lukas-h) [inline lib/src/geojson.dart:342]: Wait a second:
The original /& default constructor of the BBox has the following signature:

```dart
BBox(
    num lng1,
    num lat1,
    num alt1,
    num lng2, [
    num? lat2,
    num? alt2,
  ]) : super([
          lng1,
          lat1,
          alt1,
          lng2,
          if (lat2 != null) lat2,
          if (alt2 != null) alt2,
        ]);
```

**Review by lukas-h — CHANGES_REQUESTED** [review]

File: lib/src/geojson.dart (hunk 1 above)
Comment (lukas-h) [reply lib/src/geojson.dart:342]: We should follow the same sequence, otherwise we'll have problems

Comment (armantorkzaban) [reply lib/src/geojson.dart]: I changed the order of the optional params(Altitudes) according to standard and moved on to change the tests. Please check the result.

**Review by lukas-h — DISMISSED** [review]

**Comment by lukas-h** [comment]
2 dimensional: `[lng1, lat1, lng2, lat2]`
3 dimensional: `[lng1, lat1, alt1, lng2, lat2, alt2]`

**Review by lukas-h — APPROVED** [review]

**Pull request merged**